catalyst/utils/config.py

```diff
@@ -257,6 +258,29 @@ def parse_config_args(*, config, args, unknown_args):
     return config, args
 
 
+def get_recursive_configs(config_path):
+    configs = []
+    with open(config_path, "r") as fin:
```

Comment:
Seems that this line is redundant and can be removed

Reply:
fixed